consensus/src/block_storage/block_store.rs

```diff
@@ -100,6 +101,7 @@ pub struct BlockStore {
     time_service: Arc<dyn TimeService>,
     // consistent with round type
     back_pressure_limit: Round,
+    back_pressure_for_test: AtomicBool,
```

Comment:
can this field also be gated by feature flag?

Reply:
Is it possible to gate struct member variables through feature flag? I tried doing that but rust complains about it.

Reply:
Never mind, found a way to do it.